pay-api/migrations/versions/2024_03_06_04b8a7bed74e_ejv_link_generic.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: 04b8a7bed74e
+Revises: bacb2b859d78
+Create Date: 2024-03-06 11:40:00.153387
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '04b8a7bed74e'
+down_revision = 'bacb2b859d78'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('ejv_invoice_links', sa.Column('link_type', sa.String(length=20), nullable=True))
+    op.drop_index(op.f('ix_ejv_invoice_links_invoice_id'), table_name='ejv_invoice_links')
+    op.alter_column('ejv_invoice_links', 'invoice_id', new_column_name='link_id',
+                    existing_type=sa.Integer(), nullable=True)
+    op.rename_table('ejv_invoice_links', 'ejv_links')
+    # Update statement to set link_type to 'invoice' for all existing records as there is no data for partial refunds yet.
```

Comment:
good intent stated: "as there is no data for partial refunds yet."

Reply:
fixed

Reply:
didn't need to be fixed, was just a comment :P